health/health.go

```diff
@@ -42,12 +43,18 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, _ *http.Request) {
 	writeHealthy(w)
 }
 
-func writeUnhealthy(w http.ResponseWriter) {
-	w.Header().Set("Content-Length", "9")
+func writeHeaders(status string, w http.ResponseWriter) {
+	w.Header().Set("Content-Length", strconv.Itoa(len(status)))
```

Comment:
It's a bit unfortunate to recompute the length every time. How about passing it in as an argument to `writeHeaders` and adding

```
   const statusLen = [2 or 9]
```

in the caller?